### PR TITLE
TIP-1234: update sys reqs for PIM 4.0

### PR DESCRIPTION
**Pull request by jjanvier**

### CODE REVIEW

File: install_pim/manual/system_requirements/system_requirements.rst.inc [hunk 1]
```diff
@@ -3,9 +3,9 @@
 Akeneo PIM only works on Linux servers. The application’s behaviour has been tested on several operating systems. We cannot guarantee the behaviour of the application on servers different from:
 
 +------------------------+-------------------+
-| Debian (Linux)         | ≥ 9 (64 bits)     |
+| Debian (Linux)         | ≥ 10 (64 bits)    |
 +------------------------+-------------------+
-| Ubuntu (Linux)         | ≥ 16.04 (64 bits) |
+| Ubuntu (Linux)         | ≥ 18.04 (64 bits) |
 +------------------------+-------------------+
 | CentOS / RHEL* (Linux) | ≥ 7               |
```
Comment (jjanvier) [inline install_pim/manual/system_requirements/system_requirements.rst.inc]: should we update to RHEL 8?

File: install_pim/manual/system_requirements/system_requirements.rst.inc [hunk 2]
```diff
@@ -31,29 +31,39 @@ The web server will also need the following libraries and modules:
...
-| php7.2-zip     | No specific configuration |
+| php7.3-zip     | No specific configuration |
 +----------------+---------------------------+
-| php-exif       | No specific configuration |
+| php7.3-exif    | No specific configuration |
 +----------------+---------------------------+
-| php7.2-imagick | No specific configuration |
+| php7.3-imagick | No specific configuration |
++----------------+---------------------------+
+| php7.3-cli     | No specific configuration |
```
Comment (jjanvier) [inline install_pim/manual/system_requirements/system_requirements.rst.inc]: I added everything we use in our Docker image

File: install_pim/manual/system_requirements/system_requirements.rst.inc [hunk 3]
```diff
@@ -82,9 +92,9 @@ Besides these modules, the following configuration is the minimal configuration
 
 **Database server**
 
-+-----------------+---------------------+
-| MySQL (SQL)     | >= 5.7.22 and < 8.0 |
-+-----------------+---------------------+
++-----------------+----------+
+| MySQL (SQL)     | = 8.0.16 |
```
Comment (jjanvier) [inline install_pim/manual/system_requirements/system_requirements.rst.inc]: for now, as long as the bug https://bugs.mysql.com/bug.php?id=96823 exists
next, I'll put something like `>= 8.0.18 and < 8.1`

File: install_pim/manual/system_requirements/system_requirements.rst.inc [hunk 4]
```diff
@@ -106,15 +116,9 @@ and the following PHP modules:
 **Search engine**
 
 +---------------+------------+
-| Elasticsearch | 6.5        |
+| Elasticsearch | 7.3        |
 +---------------+------------+
 
-and the following Java Runtime Environment version
```
Comment (jjanvier) [inline install_pim/manual/system_requirements/system_requirements.rst.inc]: ES ships with its own JVM
so I guess it's totally useless to have that

File: install_pim/manual/system_requirements/system_requirements.rst.inc (hunk 1 above)
Comment (BitOne) [reply install_pim/manual/system_requirements/system_requirements.rst.inc]: Good question... That means we should test on it... Can you do a card for that? Thanks!

File: install_pim/manual/system_requirements/system_requirements.rst.inc (hunk 3 above)
Comment (BitOne) [reply install_pim/manual/system_requirements/system_requirements.rst.inc]: `>= 8.0.16 and < 8.1` then

File: install_pim/manual/system_requirements/system_requirements.rst.inc (hunk 4 above)
Comment (BitOne) [reply install_pim/manual/system_requirements/system_requirements.rst.inc]: Are you sure about that ?

Comment (BitOne) [reply install_pim/manual/system_requirements/system_requirements.rst.inc]: Okay, they bundle a JDK in the standard download, but there's an option without JDK: https://www.elastic.co/downloads/elasticsearch-no-jdk

So maybe adding a remark about that: "Elasticsearch is bundled with a JDK, but if you want to use the no JDK version, please make sure to install the right JDK for Elasticsearch 7.x"

File: install_pim/manual/system_requirements/system_requirements.rst.inc [hunk 5]
```diff
@@ -31,29 +31,39 @@ The web server will also need the following libraries and modules:
 **PHP required modules and configuration**
 
 +----------------+---------------------------+
-| PHP            | 7.2                       |
+| PHP            | 7.3                       |
 +----------------+---------------------------+
-| php-apcu       | apc.enable_cli=1          |
+| php7.3--apcu   | apc.enable_cli=1          |
```
Comment (BitOne) [inline install_pim/manual/system_requirements/system_requirements.rst.inc]: There's an extra "-"

File: install_pim/manual/system_requirements/system_requirements.rst.inc (hunk 3 above)
Comment (jjanvier) [reply install_pim/manual/system_requirements/system_requirements.rst.inc]: no because the bug is on 8.0.17 AFAIK

**Pull request merged**